src/components/App.js

```diff
@@ -57,8 +56,8 @@ class App extends React.Component {
     if (value) {
       filters.push(name);
     } else {
-      const deleteMe = filters.indexOf(name);
-      if (deleteMe >= 0) filters.splice(deleteMe, 1);
+      const i = filters.indexOf(name);
+      if (i >= 0) filters.splice(i, 1);
```

Comment:
Using single letter variables is discouraged except in certain circumstances. `deleteMe` isn't the best name, but `i` may be worse. When I see `i`, I think it's related to a loop counter but that's not what it's doing here.